src/aruku/walking.cpp

```diff
@@ -327,6 +331,124 @@ void Walking::update_orientation(double orientation)
   ORIENTATION = orientation;
 }
 
+void Walking::load_data()
+{
+  std::cout << "load_data" << std::endl;
+  std::string file_name =
+      "/home/maroqi/Projects/RobotProjects/ROS2Projects/ros2_workspace/src/aruku/config/aruku.json";
+  std::ifstream file(file_name);
+  nlohmann::json walking_data = nlohmann::json::parse(file);
+
+  for (auto &[key, val] : walking_data.items())
+  {
+    if (key == "Ratio")
+    {
+      try
+      {
+        PERIOD_TIME = static_cast<double>(walking_data[key]["period_time"].get<double>());
```

Comment:
- Remove `static_cast<double>()` if it's unnecessary.
- Instead of `get`, try to use `get_to` so the `<double>` template doesn't need to be specified as in [here](https://github.com/nlohmann/json#basic-usage).
- Is it possible to use `val` instead of using `wallking_data[key]`.